api/app/utils.py

```diff
@@ -5,7 +5,7 @@
 from sqlalchemy import func
 from notifications_utils.template import SMSMessageTemplate, PlainTextEmailTemplate
 
-local_timezone = pytz.timezone("Europe/London")
+local_timezone = pytz.timezone("Australia/Sydney")
```

Comment:
Should we move the `Australia/Sydney` timezone string to a common place like the `utils` folder?